val-ansible-role/valkey/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+valkey_version: "8.1.0"
+valkey_mode: "standalone"  
+valkey_port: 6380
+valkey_bind: "0.0.0.0"
+valkey_requirepass: ""
+valkey_cluster_nodes: []
+valkey_user: "valkey"
+valkey_install_dir: /usr/local/bin
+valkey_data_dir: "/var/lib/valkey"
+valkey_conf_dir: "/etc/valkey"
+valkey_log_file: "/var/log/valkey.log"
```

Comment:
variable defined should be like
valkey_config:
    version: "8.1.0"
    mode: "standalone"  
   port: 6380
   bind: "0.0.0.0"
   requirepass: ""
   cluster_nodes: []

Reply:
Also update related changes in tasks